src/types.js

```diff
@@ -316,7 +314,11 @@ export type Action = ActionCreators;
 export type Dispatch = ReduxDispatch<Action>;
 export type Store = ReduxStore<State, Action, Dispatch>;
 
+export type Announce = (message: string) => void;
+
 export type Hooks = {|
-  onDragStart?: (start: DragStart) => void,
-  onDragEnd: (result: DropResult) => void,
+  onDragStart?: (start: DragStart, announce: Announce) => void,
```

Comment:
all hooks now provided with a second argument: announce

This is used to announce text to screen readers